python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import argparse
+import logging
+
+#Constanst for the column names given to the query id and graded relevance.
+QUERY_ID_NAME = 'qid'
+RELEVANCE_NAME = 'relevance'
+
+
+def process_line(line, keep_additional_info):
+    if keep_additional_info == 1:
+        feature_values = line.replace('#', '').replace(' = ', ':').strip().split()
+    else:
+        feature_values = line.split('#')[0].strip().split()
+    feature_values[0] = RELEVANCE_NAME + ':' + feature_values[0]
+    r = {}
+    for fv in feature_values:
+        feat = fv.split(':')[0].strip()
+        val = fv.split(':')[1].strip()
+        if feat == RELEVANCE_NAME or feat == QUERY_ID_NAME:
+            r[feat] = val
+        else:
+            r['f_' + feat] = val
+    return r
+
+def convert(input_file, output_file, keep_additional_info, gl_2_clicks):
```

Comment:
@mohazahran Can you create an API similar to this one here? https://github.com/salesforce/ml4ir/blob/master/python/ml4ir/base/data/csv_reader.py#L14:L27